src/test/java/com/easyrestaurant/tests/ID7_TestBanOwnerAsAdmin.java

```diff
@@ -0,0 +1,53 @@
+package com.easyrestaurant.tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class ID7_TestBanOwnerAsAdmin {
+
+	@Test(priority = 1)
```

Comment:
We have 2 tests with "priority = 1". I think you need to change the priority to one of the tests to be "2".

Reply:
I changed the test to be without priority.